mmp_to_musicxml/converter.py

```diff
@@ -640,9 +650,20 @@ def convert_file(self, filepath: str):
 		score_partwise = ET.Element('score-partwise')
 
 		# title of piece
-		# TODO: allow user to set name of piece via arg
 		movement_title = ET.SubElement(score_partwise, 'movement-title')
-		movement_title.text = "title of piece goes here"
+
+		if self.opts and self.opts.title:
+			movement_title.text = self.opts.title
+			logging.debug (f"title: {movement_title.text}")
+		else:
+			movement_title.text = "title of piece goes here"
+
+		# instrument track names
+		if self.opts and self.opts.names:
+			names = self.opts.names.split(';')
+			logging.debug (f"tracks: {'|'.join(names)}")
+		else:
+			names = self.INSTRUMENTS.union(self.BASS_INSTRUMENTS)
```

Comment:
nice!

Reply:
Thanks. I'm not 100% sure the semicolon is the best list separator, though, as this will require users to enclose a list with multiple instruments in quotation marks or else escape `;` in the command-line. Commas and colons aren't a good idea either, as they're more likely to appear in the name of single instrument. How about using the plus sign `+` instead?
```
--names `violin+viola+cello`
```

Reply:
using a `+` sounds good, and yeah, feel free to add more instruments!